applications/CMakeLists.txt

```diff
@@ -45,6 +45,12 @@ add_holohub_application(endoscopy_tool_tracking DEPENDS
                                   tool_tracking_postprocessor
                                   OPTIONAL deltacast_videomaster yuan_qcap vtk_renderer)
 
+add_holohub_application(endoscopy_tool_tracking_distributed DEPENDS
+                        OPERATORS lstm_tensor_rt_inference
+                                  tool_tracking_postprocessor
+                                  OPTIONAL deltacast_videomaster yuan_qcap vtk_renderer
```

Comment:
Has the app been tested with each of these optional components? If not, would it make sense to remove them?

Reply:
No, I don't have the hardware to test them. I'll remove them.